### PR TITLE
adjust llvm and related packages

### PR DESCRIPTION
**Pull request by lovedheart**
1) Delete llvm-related packages since they are included in llvm.rb
2) Delete llvm6.rb (actually rename llvm6.rb back to llvm.rb) (no extra change for other packages that have llvm as dependency)

### CODE REVIEW

File: packages/llvm.rb [hunk 1]
```diff
@@ -1,26 +1,283 @@
...
+    uri_clang = URI.parse url_clang
+    filename_clang = File.basename(uri_clang.path)
+    sha256sum_clang = 'e07d6dd8d9ef196cfc8e8bb131cbd6a2ed0b1caf1715f9d05b0f0eeaddb6df32'
+    
+    if File.exist?(filename_clang) && Digest::SHA256.hexdigest( File.read("./#{filename_clang}") ) == sha256sum_clang
+      puts "Clang source code exists and perform unpacking".yellow
+      system "tar", "xf", "cfe-6.0.0.src.tar.xz", "-C", "tools"
+      puts "Clang source code unpacked".lightgreen
+      system "rm -rf tools/clang"  # remove possible existing folder
+      system "mv -v tools/cfe-6.0.0.src tools/clang"
```
Comment (uberhacker) [inline packages/llvm.rb]: You could move lines 38-41 outside the if statement since lines 49-51 duplicate the process.

File: packages/llvm.rb [hunk 2]
```diff
@@ -1,26 +1,283 @@
...
+    uri_lld = URI.parse url_lld
+    filename_lld = File.basename(uri_lld.path)
+    sha256sum_lld = '6b8c4a833cf30230c0213d78dbac01af21387b298225de90ab56032ca79c0e0b'
+
+    if File.exist?(filename_lld) && Digest::SHA256.hexdigest( File.read("./#{filename_lld}") ) == sha256sum_lld
+      puts "Lld source code exists and perform unpacking".yellow
+      system "tar", "xf", "lld-6.0.0.src.tar.xz", "-C", "tools"
+      puts "Lld source code unpacked".lightgreen
+      system "rm -rf tools/lld"
+      system "mv -v tools/lld-6.0.0.src tools/lld"
```
Comment (uberhacker) [inline packages/llvm.rb]: Move lines 62-65 outside the if statement and remove lines 72-75?

File: packages/llvm.rb [hunk 3]
```diff
@@ -1,26 +1,283 @@
...
+    uri_polly = URI.parse url_polly
+    filename_polly = File.basename(uri_polly.path)
+    sha256sum_polly = '47e493a799dca35bc68ca2ceaeed27c5ca09b12241f87f7220b5f5882194f59c'
+
+    if File.exist?(filename_polly) && Digest::SHA256.hexdigest( File.read("./#{filename_polly}") ) == sha256sum_polly
+      puts "Polly source code exists and perform unpacking".yellow
+      system "tar", "xf", "polly-6.0.0.src.tar.xz", "-C", "tools"
+      puts "Polly source code unpacked".lightgreen
+      system "rm -rf tools/polly"
+      system "mv -v tools/polly-6.0.0.src tools/polly"
```
Comment (uberhacker) [inline packages/llvm.rb]: Move lines 86-89 outside the if statement and remove lines 96-99?

File: packages/llvm.rb [hunk 4]
```diff
@@ -1,26 +1,283 @@
...
+    uri_compiler_rt = URI.parse url_compiler_rt
+    filename_compiler_rt = File.basename(uri_compiler_rt.path)
+    sha256sum_compiler_rt = 'd0cc1342cf57e9a8d52f5498da47a3b28d24ac0d39cbc92308781b3ee0cea79a'
+
+    if File.exist?(filename_compiler_rt) && Digest::SHA256.hexdigest( File.read("./#{filename_compiler_rt}") ) == sha256sum_compiler_rt
+      puts "Compiler-rt source code exists and perform unpacking".yellow
+      system "tar", "xf", "compiler-rt-6.0.0.src.tar.xz", "-C", "projects"
+      puts "Compiler-rt source code unpacked".lightgreen
+      system "rm -rf projects/compiler-rt"
+      system "mv -v projects/compiler-rt-6.0.0.src projects/compiler-rt"
```
Comment (uberhacker) [inline packages/llvm.rb]: Move lines 136-139 outside the if statement and remove lines 146-149?

File: packages/llvm.rb [hunk 5]
```diff
@@ -1,26 +1,283 @@
...
+    uri_libcxx = URI.parse url_libcxx
+    filename_libcxx = File.basename(uri_libcxx.path)
+    sha256sum_libcxx = '70931a87bde9d358af6cb7869e7535ec6b015f7e6df64def6d2ecdd954040dd9'
+
+    if File.exist?(filename_libcxx) && Digest::SHA256.hexdigest( File.read("./#{filename_libcxx}") ) == sha256sum_libcxx
+      puts "Libcxx source code exists and perform unpacking".yellow
+      system "tar", "xf", "libcxx-6.0.0.src.tar.xz", "-C", "projects"
+      puts "Libcxx source code unpacked".lightgreen
+      system "rm -rf projects/libcxx"
+      system "mv -v projects/libcxx-6.0.0.src projects/libcxx"
```
Comment (uberhacker) [inline packages/llvm.rb]: Move lines 161-164 outside the if statement and remove lines 171-174?

File: packages/llvm.rb [hunk 6]
```diff
@@ -1,26 +1,283 @@
...
+    uri_libcxxabi = URI.parse url_libcxxabi
+    filename_libcxxabi = File.basename(uri_libcxxabi.path)
+    sha256sum_libcxxabi = '91c6d9c5426306ce28d0627d6a4448e7d164d6a3f64b01cb1d196003b16d641b'
+
+    if File.exist?(filename_libcxxabi) && Digest::SHA256.hexdigest( File.read("./#{filename_libcxxabi}") ) == sha256sum_libcxxabi
+      puts "Libcxxabi source code exists and perform unpacking".yellow
+      system "tar", "xf", "libcxxabi-6.0.0.src.tar.xz", "-C", "projects"
+      puts "Libcxxabi source code unpacked".lightgreen
+      system "rm -rf projects/libcxxabi"
+      system "mv -v projects/libcxxabi-6.0.0.src projects/libcxxabi"
```
Comment (uberhacker) [inline packages/llvm.rb]: Move lines 186-189 outside the if statement and remove lines 196-199?

File: packages/llvm.rb [hunk 7]
```diff
@@ -1,26 +1,283 @@
...
+    uri_libunwind = URI.parse url_libunwind
+    filename_libunwind = File.basename(uri_libunwind.path)
+    sha256sum_libunwind = '256c4ed971191bde42208386c8d39e5143fa4afd098e03bd2c140c878c63f1d6'
+
+    if File.exist?(filename_libunwind) && Digest::SHA256.hexdigest( File.read("./#{filename_libunwind}") ) == sha256sum_libunwind
+      puts "Libunwind source code exists and perform unpacking".yellow
+      system "tar", "xf", "libunwind-6.0.0.src.tar.xz", "-C", "projects"
+      puts "Libunwind source code unpacked".lightgreen
+      system "rm -rf projects/libunwind"
+      system "mv -v projects/libunwind-6.0.0.src projects/libunwind"
```
Comment (uberhacker) [inline packages/llvm.rb]: Move lines 234-237 outside the if statement and remove lines 244-247?

File: packages/llvm.rb [hunk 8]
```diff
@@ -1,26 +1,283 @@
...
+    uri_openmp = URI.parse url_openmp
+    filename_openmp = File.basename(uri_openmp.path)
+    sha256sum_openmp = '7c0e050d5f7da3b057579fb3ea79ed7dc657c765011b402eb5bbe5663a7c38fc'
+
+    if File.exist?(filename_openmp) && Digest::SHA256.hexdigest( File.read("./#{filename_openmp}") ) == sha256sum_openmp
+      puts "Openmp source code exists and perform unpacking".yellow
+      system "tar", "xf", "openmp-6.0.0.src.tar.xz", "-C", "projects"
+      puts "Openmp source code unpacked".lightgreen
+      system "rm -rf projects/openmp"
+      system "mv -v projects/openmp-6.0.0.src projects/openmp"
```
Comment (uberhacker) [inline packages/llvm.rb]: Move lines 210-213 outside the if statement and remove lines 220-223?

**Comment by lovedheart** [comment]
@uberhacker I have changed the coding style.

**Pull request merged**